nixd/lib/Eval/AttrSetProvider.cpp

```diff
@@ -54,8 +54,8 @@ std::optional<Location> locationOf(nix::PosTable &PTable, nix::Value &V) {
     return std::nullopt;
 
   Position LPos = {
-      .line = static_cast<int>(NixPos.line - 1),
-      .character = static_cast<int>(NixPos.column - 1),
+      .line = static_cast<long int>(NixPos.line - 1),
```

Comment:
Do you think macos is still fine? `long int` vs `long long`?

Reply:
Why this type needs to be changed?

Reply:
Will check again. You are probably right.